100-level-AsyncAPI-Intro/intro.md

```diff
@@ -0,0 +1,19 @@
+Eve is a programming genius renowned for her ability to solve various technical challenges. She lives at 7 Optimists Lane, Capuccinova.
+
+The clock strikes 6 a.m. on a Monday, and the gentle notes of Lofi music gently rouse her from slumber.
+
+After getting herself ready and brewing a cup of coffee, Eve casts a quick glance at the clock, only to realize that she's running behind schedule for her first work meeting with Guild. Guild happens to be one of the dedicated users of AsyncAPI, the organization where Eve acts as an open-source contributor!
+
+### What is AsyncAPI?
+AsyncAPI offers a standardized approach for describing asynchronous APIs and message-driven systems through YAML or JSON formats. The AsyncAPI specification makes it easier to work with diverse messaging protocols, message structures, and event-driven architectures.
+
+The AsyncAPI specification can be defined using an AsyncAPI document, but the document itself doesn't take any direct actions or enforce rules. The real magic of the specification materializes only when an AsyncAPI document is implemented or instantiated.
```

Comment:
> The AsyncAPI specification can be defined using an AsyncAPI document,

That sentence sounds confusing to me. In fact, isn't the AsyncAPI document the one which is defined using the AsyncAPI specification 🤔 ? 
Maybe it's a language thing, not sure.

Reply:
> when an AsyncAPI document is implemented or instantiated

What does it mean to be implemented/instantiated?
Sounds like it is something programmable or executable when it is not.
Maybe we wanted to mention that the key part is the tooling maybe?

Reply:
@smoya Editing to sentences as suggested to avoid confusion in such cases. Sound good now ?